packages/devtools_app/lib/src/app.dart

```diff
@@ -8,6 +8,9 @@ import 'package:pedantic/pedantic.dart';
 import 'package:provider/provider.dart';
 
 import '../devtools.dart' as devtools;
+import 'analytics/analytics_stub.dart'
```

Comment:
i kind of like importing `as 'ga'` or `as 'analytics'`. This makes it clearer where we are making analytics calls. Here and elsewhere

Reply:
+1 to importing libs with a lot level fields with a prefix.